hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1762,20 +1762,15 @@ public List<OzoneKey> listKeys(String volumeName, String bucketName,
   @Override
   public List<RepeatedOmKeyInfo> listTrash(String volumeName, String bucketName,
       String startKeyName, String keyPrefix, int maxKeys) throws IOException {
-
-    Preconditions.checkNotNull(volumeName);
-    Preconditions.checkNotNull(bucketName);
-
-    return ozoneManagerClient.listTrash(volumeName, bucketName, startKeyName,
-        keyPrefix, maxKeys);
+    // listTrash is deprecated
+    throw new UnsupportedOperationException();
   }
 
   @Override
   public boolean recoverTrash(String volumeName, String bucketName,
       String keyName, String destinationBucket) throws IOException {
-
-    return ozoneManagerClient.recoverTrash(volumeName, bucketName, keyName,
-        destinationBucket);
+    //recoverTrash is deprecated
+    throw new UnsupportedOperationException();
```

Comment:
Similar to the previous comment, do we just mark the method with a `@Deprecated` annotation instead?

Reply:
Updated in the latest commit [b645e54](https://github.com/apache/ozone/pull/7060/commits/b645e5448bf06a866dfff844e640a6bd64053bf9)